MarsQA-CompetitionTask2/Test/Certificate_Test.cs

```diff
@@ -0,0 +1,203 @@
+﻿using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using MarsQA_CompetitionTask2.Pages.Skill;
+using MarsQA_CompetitionTask2.Utilits;
+using NUnit.Framework;
+using NUnit.Framework.Internal.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MarsQA_CompetitionTask2.Data;
+
+namespace MarsQA_CompetitionTask2.Test
+{
+    public class Certificate_Test : CommonDriver
+    {
+        Login loginobj;
+        Profile profileobj;
+        Edit editObj;
+        Delete deleteObj;
+        CertificationData CertificationData;
+        public static ExtentReports extent;
+        public static ExtentTest test;
+
+
+
+        public Certificate_Test()
+        {
+            loginobj = new Login();
+            profileobj = new Profile();
+            editObj = new Edit();
+            deleteObj = new Delete();
+
+
+        }
+        [OneTimeSetUp]
+        public static void ExtentStart()
+        {
+            //Create new instance of ExtentReports
+            extent = new ExtentReports();
+            //Create new instance of ExtentSparkReporter
+            var sparkReporter = new ExtentSparkReporter(@"C:\Users\owner\OneDrive\Documents\Mars-QA\MarsQA-CompetitionTask2\MarsQA-CompetitionTask2\Extent Reports\Certification.html");
+            //Attach the ExtentSparkReporter to the ExtentReports
+            extent.AttachReporter(sparkReporter);
+        }
+
+        [SetUp]
+        public void Login()
+        {
+
+
+            //Open Chrome Browser
+            Initialize();
```

Comment:
this is not correct you should do this in your onetimesetup and not on setup can you please fix the above and this as I think you still have not understood how hooks work! Can you also fix the other test class! Ideally you should have a Hooks calls which will create the Common Driver instance which will call the Initialize method!

Reply:
Done changes .Please review my code now.